CMakeLists.txt

```diff
@@ -189,7 +189,12 @@ endif()
 
 if(FLATBUFFERS_BUILD_SHAREDLIB)
   add_library(flatbuffers_shared SHARED ${FlatBuffers_Library_SRCS})
-  set_target_properties(flatbuffers_shared PROPERTIES OUTPUT_NAME flatbuffers)
+
+  set(FlatBuffers_Library_SONAME_MAJOR "1")
+  set(FlatBuffers_Library_SONAME_FULL "${FlatBuffers_Library_SONAME_MAJOR}.0.0")
```

Comment:
Given that the current version is 1.7.1, maybe set it to that?
Since I do releases, I'll make a note to update this with all the other versions I have to update :)

Reply:
Yes, it could be 1.7.1, I will update PR